tests/Unit/Numerical/Spectral/Test_MeanValue.cpp

```diff
@@ -0,0 +1,98 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+#include <numeric>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/IndexIterator.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "Numerical/Spectral/LegendreGaussLobatto.hpp"
+#include "Numerical/Spectral/Linearize.hpp"
+#include "Numerical/Spectral/MeanValue.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+// GCC 4.7.3 triggers the following for the CHECK() macros.
```

Comment:
Are these still needed?

Reply:
still triggered in gcc 5.3.0

Reply:
which one triggers it? We should just pop around that check, but ideally we'd have a better solution than that.

Reply:
ping